src/pages/api/card/update.js

```diff
@@ -6,9 +6,9 @@ import { withSessionRoute } from "src/lib/utils/session";
 // @access  Public
 const handler = async (req, res) => {
   try {
-    if (req.body.isOnlyComments == true) {
+    if (req.body.isOnlyNotes == true) {
```

Comment:
This was originally done to avoid the page reloading on notes changes (like it does for full card updates which happens bc of the SSR you worked on). Not sure if that was ever fixed or not.